src/docs/markdown/metrics.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Monitoring Caddy with Prometheus metrics
+---
+
+# Monitoring Caddy with Prometheus metrics
+
+Whether you're running thousands of Caddy instances in the cloud, or a single
+Caddy server on an embedded device, it's likely that at some point you'll want
+to have a high-level overview of what Caddy is doing, and how long it's taking.
+In other words, you're going to want to be able to _monitor_ Caddy.
+
+## Prometheus
+
+[Prometheus](https://prometheus.io) is a monitoring platform that collects
+metrics from monitored targets by scraping metrics HTTP endpoints on these
+targets.
```

Comment:
Maybe this could address https://github.com/caddyserver/website/pull/86/files#r493097597:

```suggestion
targets. As well as helping you to display metrics with a dashboarding tool like [Grafana](https://grafana.com/docs/grafana/latest/getting-started/what-is-grafana/), Prometheus is also used for [alerting](https://prometheus.io/docs/alerting/latest/overview/).
```